src/core/vector/qgsvectorlayerutils.cpp

```diff
@@ -1200,15 +1200,22 @@ QString QgsVectorLayerUtils::guessFriendlyIdentifierField( const QgsFields &fiel
                                   QStringLiteral( "title" ),
                                   QStringLiteral( "heibt" ),
```

Comment:
While here, you could move `heibt` under German candidates too.

Reply:
Is this german?

Reply:
> Is this german?

s/b/ß/

Reply:
> > Is this german?
> 
> s/b/ß/

Oh, the limited alphabet we Swiss live with